lib/Request.js

```diff
@@ -84,19 +84,14 @@ class Request extends CallableInstance {
     const trace = getStackLine.parse(callsites)
     const messageId = genUuid()
 
-    const originId = bubble.get('originId') || messageId
-    const bubbleId = bubble.get('bubbleId') || null
-
     const message = {
       mandatory: false,
       messageId: messageId,
       appId: this._remit._options.name,
       timestamp: now,
       headers: {
-        trace: trace,
```

Comment:
Since headers are changing do we have to be careful with migration so that we don't get queue conflicts or shouldn't they matter in this case?

Reply:
Nothing queue-wise will change so that's okay. Headers have caused issues in the past with the introduction of the old tracing metadata, but everything should be pretty confidently checking before it starts accessing things now so we should be good.